templates/addons/ddb/cf.yml

```diff
@@ -0,0 +1,24 @@
+{{.TableResourceName}}:
+  Type: AWS::DynamoDB::Table
+  Properties:
+    TableName: !Sub ${App}-${Env}-${Name}-{{.TableName}}
+    AttributeDefinitions:{{range .Attributes}}
+      - AttributeName: {{.Name}}
+        AttributeType: {{.DataType}}{{end}}
+    ProvisionedThroughput: 
+      ReadCapacityUnits: 2
+      WriteCapacityUnits: 2
+    KeySchema: 
+      - AttributeName: {{.PartitionKey}}
+        KeyType: HASH
+      - AttributeName: {{.SortKey}}
```

Comment:
I think we should add an if here, and only do the range key if someone provides it.